File: App/CountryQuiz/CountryQuiz/question.js
function CallServerMethod() {
    PageMethods.GetMessage(OnSucces);
}

function OnSucces(result) {
    alert(result);
}

function CallQuestion() {
    PageMethods.GetQuestion(OnJsonSuccess, OnJsonFail);
}

function OnJsonSuccess(result, userCtx, method) {
    if (result != null) {
        var Question = eval('(' + result + ')');

        var resultMsg = "Question called [" + method + "] is: (" + Question.QuestionTitle + "," + Question.Option1 + "," + Question.Option2 + "," + Question.Option3 + "," + Question.Option4 + ")";
        alert(resultMsg);
    }
}

function OnJsonFail(error, userCtx, method) {
    if (error != null) {
        var resultMsg = "Error from Method [" + method + "] is: '" + error.get_message() + "'";
        alert(resultMsg);
    }
}

File: App/CountryQuiz/CountryQuiz/Question.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ServiceModel;
using System.Runtime.Serialization;

namespace CountryQuiz
{
    [DataContract]
    public class Question
    {
        string question;
        string option1;
        string option2;
        string option3;
        string option4;

        [DataMember]
        public string QuestionTitle 
        {
            get { return question; }
            set { question = value; }
        }

        [DataMember]
        public string Option1 
        {
            get { return option1; }
            set { option1 = value; }
        }

        [DataMember]
        public string Option2
        {
            get { return option2; }
            set { option2 = value; }
        }

        [DataMember]
        public string Option3
        {
            get { return option3; }
            set { option3 = value; }
        }

        [DataMember]
        public string Option4
        {
            get { return option4; }
            set { option4 = value; }
        }

        public Question(string q, string o1, string  o2, string o3, string o4) 
        {
            QuestionTitle = q;
            Option1 = o1;
            Option2 = o2;
            Option3 = o3;
            Option4 = o4;
        }
    }
}